test-repos/downloadTestRepos.nim

```diff
@@ -0,0 +1,30 @@
+import std/[os, strutils]
+
+proc exec*(cmd: string) =
+  if execShellCmd(cmd) != 0:
+    quit "FAILURE: " & cmd
+
+template withDir*(dir: string; body: untyped) =
+  let old = getCurrentDir()
+  try:
+    setCurrentDir(dir)
+    echo "WITHDIR: ", dir, " at: ", getCurrentDir()
+    body
+  finally:
+    setCurrentDir(old)
+
+when isMainModule:
+  withDir "test-repos":
+    let host = "https://github.com"
+    let repo = "elcritch/atlas"
+    let release = "releases/download/test-repos-v0.8.0"
```

Comment:
```suggestion
    let repo = "nim-lang/atlas"
    let release = "releases/download/test-repos-v0.8.0"
```

These lines will need to be changed. Either we can setup a release for the zip file now, or do it after. 

Here's what my release looked like:

https://github.com/elcritch/atlas/releases/tag/test-repos-v0.8.0

Reply:
Alternatively, turn it into it's own repo.

Reply:
> Alternatively, turn it into it's own repo.

Sounds like the best option to me.

Reply:
Cool, that’d be easy to setup. 

Could you create a new repo and ping me? I’ll make up a PR for it.